src/sentry/integrations/vercel/uninstall.py

```diff
@@ -60,35 +92,33 @@ def delete(self, request):
             integration.metadata["webhook_id"] = next_config["webhook_id"]
             integration.metadata["installation_id"] = next_config_id
 
+        integration.save()
+
+        # At this point we can consider if len(orgs) > 1. We have already updated the
+        # integration.metadata, but we may not have deleted the OrganizationIntegration
         try:
             OrganizationIntegration.objects.get(
                 organization_id=configuration["organization_id"], integration_id=integration.id
             ).delete()
+
+            organization = Organization.objects.get(id=configuration["organization_id"])
+            create_audit_entry(
```

Comment:
@MeredithAnya Seems like you could define a function to call `create_audit_entry` that takes the `organization` as an input and uses closure for `integration`. That way, you don't have to have so much duplicated code.